### ISSUE TITLE
Auditing allowed operations

### ISSUE THREAD

**Issue by nam-jaehyun** [kubearmor/KubeArmor#64]
KubeArmor provides three types of actions: Allow, Audit, and Block.

The current purpose of the 'Audit' action is to check if some policies are properly defined before using the 'Block' action.
So, we can basically get alerts related to certain security rules with the 'Audit' and 'Block' actions.

How about the 'allow' action? We can get alerts for some objects or operations that are not allowed.
In other words, there is no way to get some logs for exactly allowed access or operations.

Thus, we'll add a new action called 'AllowWithAudit'.
